### DISCUSSION TITLE
Trouble loading venus-os_dbus-enphase-envoy

### DISCUSSION THREAD

**Discussion by hahnpremium** [mr-manuel/venus-os_dbus-enphase-envoy#11] [Q&A]
Hello, I am very keen to load this onto my multiplus-II, but it seems my ambition has outweighed my talent, I realise I'm must be missing some important steps, or removing the semi colons in the wrong locations.  If someone would be kind enough to give me advice / walk through, anything would be appreciated.  Please see my poor attempt at loading the config file and then adding it in the VRM console.

FYI - I have a Enphase IQ8 3 phase system running Software Version 8.2.127, with both production and consumption CT's, With a Victron Multiplus-II GX ESS system single phase.  I'd like to firstly, communicate the inverter data to the VRM and if I could use the Enphase CT's for production and consumption metering.

Thank you in advance for any help anyone can offer and for making this code.

![Config File Enphase Venus OS](https://github.com/user-attachments/assets/6e21213c-bd98-495e-9aaf-a29a7c4f255b)

![Log file](https://github.com/user-attachments/assets/0d900456-a1b7-41b5-a441-dcde76a36d6e)

**Comment by mr-manuel** [comment]
I don't know how you did it, but you broke completely the config file. If you are not familiar with it start over or compare it with the original one. Lots of semicolons are missing before comments.

**Reply by hahnpremium** [reply]
I don't even know what I did, but I restarted clean, input the IP, login, gateway serial no etc. and saved config.ini (edited) to the correct spot in the dbus-enphase-envoy (I failed at this step aswell).  Worked straight away, same with the MQTT data stream.  Thanks for writing this, it ties my system together nicely.

**Reply by mr-manuel** [reply]
You're welcome!

Feel free to [buy me one or more beers](https://www.paypal.com/donate/?hosted_button_id=3NEVZBDM5KABW), if you like my scripts :-)
